base/extras.wad/sbardef.lmp

```diff
@@ -0,0 +1,2002 @@
+{
```

Comment:
Oh, wow, this is taking dimensions. Why are these extra lumps necessary?

Reply:
SBARDEF cannot be updated as DEHACKED/UMAPINFO, only the last one in the load order is used.

id1.wad has the FTNKA0 sprite in SBARDEF, but does not have it itself (it is taken from id24res.wad). Also the wide layout won't work, that's our addition. Same with extras.wad (we want to autoload extras.wad because of the remixed soundtrack).

Reply:
Should we merge this?